std/system.d

```diff
@@ -82,5 +82,76 @@ immutable
     /// The endianness that the program was compiled for.
     version (LittleEndian) Endian endian = Endian.littleEndian;
     else                  Endian endian = Endian.bigEndian;
+    /++
+        CPU architecture.
+
+        Note:
+            This is intended for cases where you need a value representing the
+            CPU architecture at runtime. If you're doing something which should
+            compile differently depending on CPU architecture, then please use
+            `version (X86_64)`, `version (ARM)`, etc.
+
+        See_Also:
+            $(DDSUBLINK spec/version,PredefinedVersions, Predefined Versions)
+      +/
+    enum CPU
+    {
+        x86,   /// Intel and AMD 32-bit processors
+        x86_64, /// Intel and AMD 64-bit processors
+        arm, /// The ARM architecture (32-bit) (AArch32 et al)
+        aarch64, /// The Advanced RISC Machine architecture (64-bit)
+        asmJS, /// The asm.js intermediate programming language
+        avr, /// 8-bit Atmel AVR Microcontrollers
+        epiphany, /// The Epiphany architecture
+        ppc, /// The PowerPC architecture, 32-bit
+        ppc64, /// The PowerPC architecture, 64-bit
```

Comment:
We (by which I probably mean Iain habitually given that he actually has the ability to test on one) habitually confuse IBM Power and IBM PowerPC. one is dead the other is not

Reply:
I simply used the names and descriptions from the [spec page](https://dlang.org/spec/version.html#predefined-versions). If those are inaccurate, someone more familiar with the matter should correct it...

Reply:
Just call it rs6000 then ;-)